cmd/ebctl/sboms.go

```diff
@@ -225,6 +226,15 @@ func (cli *CLI) inferSBOMInfo(ctx context.Context, sbomData []byte) (*inferredSB
 				sbomInfo.ImageTag = metadata.Tags[0]
 			}
 		}
+
+	case platform.SBOMFormat_SBOM_FORMAT_CYCLONEDX_JSON:
+		metadata, ok := sbom.Source.Metadata.(source.StereoscopeImageSourceMetadata)
+		if ok {
+			sbomInfo.ImageID = metadata.ID
+			if len(metadata.Tags) > 0 {
+				sbomInfo.ImageTag = metadata.Tags[0]
+			}
+		}
```

Comment:
This is identical (right?) to the one above. You can use `case foo, bar:` syntax